server/src/main/java/org/opensearch/monitor/fs/FsInfo.java

```diff
@@ -211,8 +211,8 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
             if (fileCacheReserved != -1) {
                 builder.humanReadableField(Fields.CACHE_RESERVED_IN_BYTES, Fields.CACHE_RESERVED, getFileCacheReserved());
             }
-            if (fileCacheReserved != 0) {
-                builder.humanReadableField(Fields.CACHE_UTILIZED, Fields.CACHE_UTILIZED_IN_BYTES, getFileCacheUtilized());
+            if (fileCacheUtilized != 0) {
```

Comment:
@lukas-vlcek could we add a test for it please? thank you

Reply:
@reta yes, always a good idea!

Reply:
I actually dislike these conditions `if (fileCacheReserved != -1)` and `if (fileCacheUtilized != 0)` and I am about to open a new/separate ticket about these... but yes, let's see if we can have at least some test.

Reply:
@reta, resolved. I am going to update issue description and add some more explanation about what was needed to do to make it possible to test (and what I do not like about it).